asapdiscovery-data/asapdiscovery/data/scripts/sdf_to_smiles_list.py

```diff
@@ -0,0 +1,36 @@
+"""
+Load in a multi-ligand SDF file and write out a txt file with the format `SMILES <space> ligand_name`
+Ligand name expected to be in the Compound_ID field of the sdf file.
+"""
+from argparse import ArgumentParser
+
+
+def get_args():
+    parser = ArgumentParser(description="")
+    parser.add_argument(
+        "-s",
+        "--sdf_fn",
+        required=True,
+        help="SDF file containing molecules of interest.",
+    )
+    parser.add_argument("-o", "--output_fn", required=True, help="Output file.")
+    return parser.parse_args()
+
+
+def main():
+    args = get_args()
+    from asapdiscovery.data.openeye import load_openeye_sdfs, oechem
```

Comment:
move imports to the top

Reply:
done https://github.com/choderalab/asapdiscovery/pull/105/commits/0df4dd6831b443ddf0b1ec6c37eb24cf8fc7085d